package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "azuredatastudio",
   "version": "1.31.0",
-  "distro": "462bd5082b18c2ed5df4ba615d3dda9a2860a394",
+  "distro": "d69a2de3200adff1ccbccd53a6dae7074f883bb5",
```

Comment:
I've added some changes to the release branch for web build that you'll likely need to pickup to get a clean build.  I think the commit hash `9eab1e6d6180c1a560a4a50eb9e6e99641549329` should be fine.

Reply:
Thanks, sorry about causing you problems with the web mode stuff - I want to follow up and figure out if we can make it so we don't have to update the distro when adding new smoke tests if I can to hopefully avoid this kind of thing in the future!